Common.Configurations.targets

```diff
@@ -33,7 +33,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.25" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.27" />
+    <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.4" />
```

Comment:
Update version

Reply:
i can't now do it and for what? you don't fix models which i use here